docs/v2/spec/2018_Public_TS_Schema.csv

```diff
@@ -0,0 +1,13 @@
+﻿Field Name,Field Data Type,Valid Values,Descriptions and Examples

```

Comment:
Can you rename this file from Schema to data_dictionary per our conversation today?